common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerOAuth2TokenCache.java

```diff
@@ -1685,7 +1711,7 @@ public void setSingleSignOnState(@NonNull final String uidStr,
 
         MsalOAuth2TokenCache targetCache;
 
-        final int uid = Integer.valueOf(uidStr);
+        final int uid = Integer.parseInt(uidStr);
```

Comment:
This is actually a great change.  It'll result in one fewer auto boxed Integer being created and destroyed - however did you find it?

Reply:
It's all thanks to spotbugs lol